addons/website_livechat/static/src/components/visitor_banner/visitor_banner.scss

```diff
@@ -0,0 +1,90 @@
+// -----------------------------------------------------------------------------
+// Layout
+// -----------------------------------------------------------------------------
+
+.o_VisitorBanner {
+    border-bottom-width: 1px;
+    border-bottom-style: solid;
+    display: flex;
+    flex: 0 0 auto;
+    padding: map-get($spacers, 4) map-get($spacers, 2);
+}
+
+.o_VisitorBanner_avatar {
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+}
+
+.o_VisitorBanner_avatarContainer {
+    height: 36px;
+    margin-left: map-get($spacers, 1);
+    margin-right: map-get($spacers, 1);
+    position: relative;
+    width: 36px;
+}
+
+.o_VisitorBanner_country {
+    margin-inline-end: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_history {
+    margin-top: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_historyIcon {
+    margin-inline-end: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_language {
+    margin-inline-end: map-get($spacers, 3);
+}
+
+.o_VisitorBanner_languageIcon {
+    margin-inline-end: map-get($spacers, 1);
+}
+
+.o_VisitorBanner_onlineStatusIcon {
+    @include o-position-absolute($bottom: 0, $right: 0);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-flow: column;
+    width: 1.2em;
+    height: 1.2em;
+    line-height: 1.3em;
```

Comment:
Where are those coming from?

Shouldn't this be somehow related to/using `PartnerImStatusIcon`?

Reply:
It's copied from there yes, I know there is a task to refactor the user icons so I didn't dig into it

Reply:
(Talking about this one : https://www.odoo.com/web#id=2282361&action=333&active_id=1750&model=project.task&view_type=form&cids=1&menu_id=4720 )

Reply:
ok to keep a copy here, I'll add this use case in the task